rotkehlchen/externalapis/opensea.py

```diff
@@ -210,14 +219,29 @@ def _deserialize_nft(
                 if last_sale['payment_token']['symbol'] in {'ETH', 'WETH'}:
                     payment_asset = self.eth_asset
                 else:
-                    payment_asset = Asset(
-                        ethaddress_to_identifier(
-                            to_checksum_address(last_sale['payment_token']['address']),
-                        ),
-                    ).resolve_to_evm_token()
-
-                amount = asset_normalized_value(int(last_sale['total_price']), payment_asset)
-                eth_price = FVal(last_sale['payment_token']['eth_price'])
+                    payment_asset = get_or_create_evm_token(
+                        userdb=self.db,  # type: ignore  # mypy doesn't detect that self.db is not None
+                        evm_address=to_checksum_address(last_sale['payment_token']['address']),
```

Comment:
This should use `deserialize_evm_address`. Otherwise this will raise `ValueError` in case of error